src/Microsoft.Data.Analysis/DataFrameWindow.Rolling.PrimitiveDataFrameColumn.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Data.Analysis
+{
+    /// <summary>
+    /// A window for <see cref="PrimitiveDataFrameColumn{T}"/> to support rolling window operations
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public class PrimitiveDataFrameColumnRollingWindow<T> : DataFrameColumnWindow
```

Comment:
The file name and the class name should match. That way someone can fine the file a class is in